pages/linux/upnpc.md

```diff
@@ -0,0 +1,20 @@
+# upnpc
+
+> Configure port forwarding rules on your router via the UPnP protocol.
+> More information: <http://miniupnp.free.fr/>.
+
+- Forward external TCP port 80 to port 8080 on a local machine:
```

Comment:
```suggestion
- Forward an external TCP port 80 to port 8080 on a local machine:
```

Reply:
Respectfully, I disagree with this edit. There is only one port number 80, THE port 80, so the "an" article is not necessary.